helpers/mail/mail_v3_test.go

```diff
@@ -63,7 +63,7 @@ func TestV3AddPersonalizations(t *testing.T) {
 	m.AddPersonalizations(personalizations...)
 
 	if len(m.Personalizations) != numOfPersonalizations {
-		t.Errorf("Mail should have %d personalizations, got %d personalizations", personalizations, len(m.Personalizations))
+		t.Errorf("Mail should have %d personalizations, got %d personalizations", len(personalizations), len(m.Personalizations))
```

Comment:
Here you use len(personalizations)

but on lines 81 and 96, it looks like you used the numOf* variable. Which is preferrable?

Reply:
Sorry @mbernier didn't check this review comment. Seems the changes are now merged. Will clean this up in later PR.